amplify/backend/function/updateContributorsWithReputation/src/utils.js

```diff
@@ -281,6 +282,29 @@ const updateReputationInDomain = async ({
   );
 };
 
+const calculatePercentageReputation = (userReputation, totalReputation) => {
+  if (!userReputation || !totalReputation) return null;
+  const userReputationNumber = BigNumber.from(userReputation);
+  const totalReputationNumber = BigNumber.from(totalReputation);
+
+  const reputationSafeguard = BigNumber.from(100).pow(2);
+
+  if (
+    userReputationNumber.isZero() ||
+    totalReputationNumber.isZero() ||
+    userReputationNumber.mul(reputationSafeguard).lt(totalReputationNumber)
+  ) {
+    return 0;
```

Comment:
Thank you for keeping the fallback here.

I've put this in a while back when I noticed colony rep percentage was not coming in during updates, breaking this logic and setting it to empty in the database

This ultimately broke the members/reputation query